web-src/src/services/FirefallService.js

```diff
@@ -28,20 +28,22 @@ export class FirefallService {
     console.debug(`Feedback: ${this.feedbackEndpoint}`);
   }
 
-  async complete(prompt, temperature) {
+  async complete(prompt, temperature, asJson) {
```

Comment:
If we never ask for text responses, there's no need to include this parameter here.

Reply:
We do need text responses for text-to-image prompt suggestions.